zebra-chain/src/transaction.rs

```diff
@@ -628,6 +628,45 @@ impl Transaction {
         }
     }
 
+    /// Returns the Sprout note commitments in this transaction.
+    pub fn sprout_note_commitments(
```

Comment:
I just moved this fn into the Sprout section in the source code. It was in the section for Sapling previously.